projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -10,7 +10,7 @@
     "cy:open:epd-visualization": "cypress open --env BASE_SITE=powertools-epdvisualization-spa",
     "cy:run": "cypress run",
     "cy:run:b2b": "cypress run --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers",
-    "cy:run:core": "cypress run --spec \"cypress/integration/**/*.core-e2e-spec.ts\"",
+    "cy:run:core": "cypress run --reporter junit --reporter-options mochaFile=results/test-results-[hash].xml --spec \"cypress/integration/**/*.core-e2e-spec.ts\"",
```

Comment:
I don't think this is part of this PR since this is for unit tests only and the other PR is for cypress right? If so, please remove

Reply:
correct, will be removed